app/sensors/DropdownList.js

```diff
@@ -376,7 +372,7 @@ export default class DropdownList extends Component {
 							<Select
 								options={this.state.items}
 								clearable={false}
-								value={this.state.value}
+								value={_.isArray(this.state.value) ? this.state.value.join() : this.state.value}
```

Comment:
Again, you can use the `multipleSelect` prop here to check for `MultiDropdownList` (which should always have array values).